packages/modelserver-client/src/model-server-client-v2-integration.spec.ts

```diff
@@ -234,7 +240,7 @@ describe('Integration tests for ModelServerClientV2', () => {
             const notification = await patchNotification;
             const patch = notification.patch;
 
-            expect(notification.modelUri).to.be.equal(modeluri);
+            expect(notification.modeluri.equals(modeluri)).to.be.true;
```

Comment:
My only quibble with this is that it loses the presentation of the string comparison when the test fails, which is highly informative in the test report.

Is there a good reason not to test equality of the toString() conversions of the URIs? I wouldn't go so far as to implement a Chai assertion just for equality testing of URIs ... 😀 

(there are many more occurrences of this throughout the tests)

Reply:
Looking at the [docs](http://medialize.github.io/URI.js/docs.html#equals), the equals method does things like normalization and also ignores the order of query parameters. This is probably not needed here in the tests, but still cleaner imho.
The `expect` accepts a second parameter, which we should use to provide an error message which is helpful.
@cdamus, @ndoschek what do you think?

Reply:
why not use the second parameter for a better failure error?

Reply:
Oh sorry, I didn't see your comment earlier. As discussed we will stick to the string comparisons for now and will create dedicated URI tests in a follow up (see #123 ) where we should test all varieties (e.g. relative hierarchies, random order of query parameters).